docs/getting-started/server/database/ef/index.mdx

```diff
@@ -0,0 +1,215 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Entity Framework
+
+:::warning
+
+Entity Framework support is still in beta and is not suitable for production databases.
+
+:::
+
+:::info
+
+This page refers to setting up a Bitwarden instance to develop on, for instructions on testing out
+our EF deployments for personal use, such as Bitwarden Unified, please see
+[the help documentation](https://bitwarden.com/help/install-and-deploy-unified-beta/)
+
+:::
+
+## Background
+
+Entity Framework (EF) is an ORM framework that acts as a wrapper around a database. It allows us to
+support multiple (non-MSSQL) databases without having to maintain migration and query scripts for
+each.
+
+Our EF implementations currently support Postgres, MySQL, and SQLite3.
+
+## Setting up EF databases
```

Comment:
ℹ️ As I mentioned to @Hinton in a prior PR here let's be consistent on title casing.